components/Guide/Guide.js

```diff
@@ -80,6 +80,15 @@ class Guide extends Component {
     );
   }
 
+  get contentComponent() {
+    const { content } = this.props;
+    if (typeof content === 'string') {
```

Comment:
This case should be the default case. The logic should be:
if is react component then render it, else render as a markdown component.